assets/pages/showcase/showcase.scss

```diff
@@ -0,0 +1,213 @@
+// -----  gu-sass ----- //
+
+@import '~stylesheets/gu-sass/gu-sass';
+
+// ----- Shared Components ----- //
+@import '~components/footer/footer';
+@import '~components/headers/simpleHeader/simpleHeader';
+
+@import '~components/svgs/svg';
+
+@import '~components/headingBlock/headingBlock';
+@import '~components/leftMarginSection/leftMarginSection';
+@import '~components/productPage/productPageHero/productPageHero';
+@import '~components/productPage/productPageContentBlock/productPageContentBlock';
+@import '~components/productPage/productPageTextBlock/productPageTextBlock';
+@import '~components/productPage/productPageButton/productPageButton';
+@import '~components/productPage/productPageFeatures/productPageFeatures';
+
+:root {
+  scroll-behavior: smooth;
+}
+
+#showcase-landing-page {
+  .component-product-page-hero {
+    @include mq($from: tablet) {
+      .component-heading-block {
+        margin-top: 24em;
+      }
+      .component-grid-picture {
+        position: absolute;
+        top: 0;
+        min-width: 100%;
+        height: 100%;
+        @include mq($from: wide) {
+          left: gu-span(2) * -1;
+        }
+      }
+      .component-grid-picture__image {
+        height: 100%;
+        display: block;
+      }
+    }
+    @include mq($until: tablet) {
+      .component-grid-picture {
+        max-height: 20em;
+        overflow: hidden;
+        display: block;
+      }
+      .component-grid-picture__image {
+        max-width: gu-span(8);
+        margin: auto;
+        float: none;
+        display: block;
+      }
+    }
+  }
+
+  .anchor {
+    color: gu-colour(header-dark);
+    font-family: $gu-headline;
+    font-size: 26px;
+    font-weight: bold;
+    line-height: 1.12;
+    padding: ($gu-v-spacing / 2) 0 ($gu-v-spacing * 1.5);
+
+    @include mq($from: tablet) {
+      font-size: 30px;
+      width: 75%;
+    }
+
+    @include mq($from: desktop) {
+      font-size: 36px;
+      width: 75%;
+    }
+  }
+
+  .component-product-page-content-block {
+    &--grey {
+      @include mq($from: tablet) {
+        .svg-beyond-headlines,
+        .svg-no-one {
+          margin-left: $gu-v-spacing * -6;
+          margin-right: $gu-v-spacing * -6;
```

Comment:
It might be worth just using `-72px` here (I think that's what this evaluates to)? `$gu-v-spacing` is typically for vertical spacing.

Reply:
alternatively, i'd use a `$gu-h-spacing` equivalent

Reply:
updated to `$gu-h-spacing`